roles/kubernetes/node/tasks/install.yml

```diff
@@ -25,6 +25,20 @@
   template: "src=kubelet.{{ kubelet_deployment_type }}.service.j2 dest=/etc/systemd/system/kubelet.service backup=yes"
   notify: restart kubelet
 
+- name: install | Set SSL CA directories
+  set_fact:
+    ssl_ca_dirs: "[
+      {% if ansible_os_family in ['CoreOS', 'Container Linux by CoreOS'] -%}
+      '/usr/share/ca-certificates',
+      {% elif ansible_os_family == 'RedHat' -%}
+      '/etc/pki/tls',
+      '/etc/pki/ca-trust',
+      {% elif ansible_os_family == 'Debian' -%}
+      '/usr/share/ca-certificates',
+      {% endif -%}
+    ]"
+  tags: facts
+
```

Comment:
As far as I can tell, the fact ``ssl_ca_dirs`` is only used by the template ``kubelet-container.j2`` called in the task below. I moved it here out of gen_certs_script.yml, to avoid errors when gen_certs_vault.yml is used. I never saw this locally because we use ``kubelet_deploy_type == "rkt"``.